test/controllers/records_controller_test.rb

```diff
@@ -154,6 +154,69 @@ class RecordsControllerTest < ActionController::TestCase
       assert_equal 'The record C1000000020-LANCEAMSR2 was successfully updated.', flash[:notice]
       assert_equal Record.find(18).state, 'ready_for_daac_review'
     end
+
```

Comment:
```
      user = User.find_by(email: 'arc_curator@element84.com')
      sign_in(user)
      stub_urs_access(user.uid, user.access_token, user.refresh_token)
      record = Record.find(1)
      post :associate_granule_to_collection, params: { id: record.id, associated_granule_value: 16 }
      assert_equal 'Granule G309210-GHRC/6 has been successfully associated to this collection revision 8. ', flash[:notice]
```
Can be moved to a shared before block.

Reply:
Take a look at the latest nested workflow and tell me if this comment you made still makes sense.  not sure exactly what you mean with the latest.